### PR TITLE
fix: For some issues

### PR DESCRIPTION
**Pull request by percebus**
# Issues

- https://github.com/percebus/azure-secure-networking-for-devs/issues/43
- https://github.com/percebus/azure-secure-networking-for-devs/issues/36
- https://github.com/percebus/azure-secure-networking-for-devs/issues/33
- https://github.com/percebus/azure-secure-networking-for-devs/issues/32
- https://github.com/percebus/azure-secure-networking-for-devs/issues/27
- https://github.com/percebus/azure-secure-networking-for-devs/issues/24
- https://github.com/percebus/azure-secure-networking-for-devs/issues/23
- https://github.com/percebus/azure-secure-networking-for-devs/issues/16

### CODE REVIEW

File: docs/tutorial/03/README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ If you haven't already, please go through the following:
    1. Your laptop (Optional)
    1. In the JumpBox VM
 
+> [!TIP]
+> REMINDER: Is the Jumpbox VM running?
+
+Sometimes the VM shuts down.
```
Comment (percebus) [inline docs/tutorial/03/README.md:34]: https://github.com/percebus/azure-secure-networking-for-devs/issues/43

File: docs/tutorial/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Don't have to worry about public IP address changes that might require config changes on your part
+- Simplified access for remote users
+
+#### Cost
+
+> "How expensive is private link?"
+
+[Azure Private Link pricing](https://azure.microsoft.com/en-us/pricing/details/private-link/)
+
+##### As of 2024-11-22
```
Comment (percebus) [inline docs/tutorial/FAQ.md:30]: https://github.com/percebus/azure-secure-networking-for-devs/issues/32

File: docs/tutorial/02/hub/rt.md [hunk 3]
```diff
@@ -6,10 +6,51 @@
    1. Going to the internet: through the **Azure Firewall**
    1. Going to other VNets
 
+## Before you begin
+
+### Routing
```
Comment (percebus) [inline docs/tutorial/02/hub/rt.md:11]: https://github.com/percebus/azure-secure-networking-for-devs/issues/27

File: assets/img/azure/solution/vnets/hub/vnet/create/security.png
Comment (percebus) [inline assets/img/azure/solution/vnets/hub/vnet/create/security.png:1]: https://github.com/percebus/azure-secure-networking-for-devs/issues/23

**Review by ivansholokh — APPROVED** [review]

**Pull request merged**